core/arch/x86/instr_create.h

```diff
@@ -1751,6 +1769,28 @@
     instr_create_1dst_2src((dc), OP_aeskeygenassist, (d), (s), (i))
 /* @} */ /* end doxygen group */
 
+/* 1 destination, 2 sources */
+/** @name 1 destination, 1 mask, and 2 sources */
+/* @{ */ /* doxygen start group; w/ DISTRIBUTE_GROUP_DOC=YES, one comment suffices. */
+/**
+ * This INSTR_CREATE_xxx macro creates an instr_t with opcode OP_xxx and the given
+ * explicit operands, automatically supplying any implicit operands.
+ * \param dc The void * dcontext used to allocate memory for the instr_t.
+ * \param d The opnd_t explicit destination operand for the instruction.
+ * \param k The opnd_t explicit mask source operand for the instruction.
+ * \param i The opnd_t explicit second source operand for the instruction, which
+ * \param s The opnd_t explicit source operand for the instruction.
+ * must be an immediate integer (opnd_create_immed_int()).
```

Comment:
dangling sentence end -- probably goes with the "which" above

Reply:
Yes, see above, fixed.